python/dgl/sparse/sparse_matrix.py

```diff
@@ -586,6 +586,93 @@ def range_select(self, dim: int, index: slice):
             )
         raise TypeError(f"{type(index).__name__} is unsupported input type.")
 
+    def sample(
+        self,
+        dim: int,
+        n_pick: int,
```

Comment:
n_pick -> fanouts

Reply:
GNN sampling does not have the case that different nodes sample different numbers of neighbors. So an integer `fanout` should be ok.

Reply:
SGTM